examples/guides/se-restful-webservice/README.adoc

```diff
@@ -642,31 +547,31 @@ These link the health-related endpoints your code is exposing to the new methods
 
 // tag::tryReadiness[]
 
-=== Try the readiness check
-Access the readiness check endpoint:
+=== Check the server's health
+Run this command
 [source,bash]
-curl -i -X GET http://localhost:8080/ready
+curl -i -X GET http://localhost:8080/health
 
-Our readiness check returns no payload, just the 200 status, so you won't see any data
-displayed. The `-i` option shows the
-200 status in the response.
-// end::tryReadiness[]
-
-// tag::tryLiveness[]
+and you should see output like this
+[listing,subs=+quotes]
+----
+{"outcome":"UP","checks":[{"name":"deadlock","state":"UP"},{"name":"diskSpace","state":"UP","data":{"free":"180.64 GB","freeBytes":193955860480,"percentFree":"38.79%","total":"465.72 GB","totalBytes":500068036608}},*{"name":"greetingAlive","state":"UP"}*,{"name":"heapMemory","state":"UP","data":{"free":"230.86 MB","freeBytes":242074232,"max":"4.00 GB","maxBytes":4294967296,"percentFree":"99.39%","total":"256.00 MB","totalBytes":268435456}}]}
+----
 
-=== Try the liveness check
-. Ping the health check endpoint
-+
---
-Without changing the greeting, ping the health endpoint:
-[source,bash]
-curl -i -X GET http://localhost:8080/alive
+// tag::se-HealthChecks-notes[]
+The JSON output conveys various health indicators because the generated code
+included `HealthChecks.healthChecks()` in the `HealthSupport.builder`.
+// end::se-HealthChecks-notes[]
+The first item is `outcome` which describes the overall health of the
+server based on all the other indicators. All indicators are `UP` so the outcome
+is as well. You should also see our app-specific liveness check in the output
```

Comment:
The `state` of all the indicators is `UP`. So the `outcome` field shows `UP`.

Reply:
Consider mentioning what value `outcome` will show if the `state` field for one or more indicators is not `UP`.

Reply:
done